mandatory/2-function-creation.js

```diff
@@ -15,37 +21,93 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice>200){
+    return totalPrice-(totalPrice * 0.1)
+  }else {
+    return totalPrice-(totalPrice *0.05)
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+
+  function printOddNumbers(limit) {
+    let numberOfLoops= 1
+  while (numberOfLoops <limit) {
+    console.log(numberOfLoops);
+    numberOfLoops = numberOfLoops + 2;
+}
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+
+
+function buyTwoGetTheCheapestFree(price1, price2) {
+//   let price1 = 700
+// let price2 = 500
+if(price1>price2) {
+  return price1;
+}else if (price1 < price2)
+  return price2;
+}
+
+
 
 /*
   Complete the function to determine if it is suitable for a person to register based on their age!
   - if the person is 12 or younger it should return "You Are Too Young To Register"
   - if the person is older than 12 and younger than 90 it should return "You Can Register"
   - if the person is 90 or older it should return "You Don't Need To Register"
 */
-function canRegister(age) {}
+function canRegister(age) {
+if(age<=12){
+return "You Are Too Young To Register"
+}
+else if (age> 12 && age<90){
+return "You Can Register"
+}else {
+  return "You Don't Need To Register"
+}
+}
```

Comment:
Comment: A stylistic thing. Sometimes you have spaces between code blocks and sometimes not. Compare Line 68 (age<12){
But on line 71 (age> 12 && age<90){